src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -956,8 +956,10 @@ public Keyworker getKeyworkerByOffenderNo(@PathVariable("offenderNo") @ApiParam(
             @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class)})
     @ApiOperation(value = "The next visit for the offender.", notes = "The next visit for the offender.", nickname = "getBookingVisitsNext")
     @GetMapping("/{bookingId}/visits/next")
-    public VisitDetails getBookingVisitsNext(@PathVariable("bookingId") @ApiParam(value = "The offender booking id", required = true) final Long bookingId) {
-        return bookingService.getBookingVisitNext(bookingId);
+    public VisitDetails getBookingVisitsNext(
+        @PathVariable("bookingId") @ApiParam(value = "The offender booking id", required = true) final Long bookingId,
+        @RequestParam(value = "withVisitors", required = false, defaultValue = "false") @ApiParam(value = "Toggle to return Visitors in response (or not).", required = true) final boolean withVisitors) {
```

Comment:
think this should be `required = false` in `ApiParam`

Reply:
fixed that